### ISSUE TITLE
Visible <div> tags in writePlain

### ISSUE THREAD

**Issue by t-8ch** [jgm/pandoc#3178]
When creating a naive html to text converter the plain text output contains unexpected `<div>` items.
(This was different in the past, not sure when)
Removing the the `raw_html` extension fixes it.
The CLI does not use the default options when writing plain text, but a reduced subset, which does not include this extension, so it does not exhibit this behaviour.
Unfortunately the set of default extensions for the CLI (`Text.Pandoc.getDefaultExtensions`) are not exported.

``` haskell
-- test.hs
import Data.List
import Data.Set

import Text.Pandoc
import Text.Pandoc.Error

htmlToText :: String -> String
htmlToText = writePlain def . handleError . readHtml def

htmlToTextExpected :: String -> String
htmlToTextExpected = writePlain (def {
    writerExtensions = Data.Set.filter (/= Ext_raw_html) (writerExtensions def)
  }) . handleError . readHtml def

testcases = ["<div>foo</div>", "<span>bar</span>"]

main :: IO ()
main = putStrLn $ intercalate "\n---------\n" $ concatMap
    (\testcase -> [htmlToText testcase, htmlToTextExpected testcase]) testcases
```

``` sh
$ runhaskell test.hs
<div>

foo

</div>
---------
foo
---------
bar
---------
bar
```

Tested with 1.17.2 and latest HEAD (738806)

**Comment by jgm** [comment]
You need to set `writerExtensions` to `plainExtensions`, in addition to using `writePlain`.
(I know, it's not super-intuitive.)  See `getDefaultExtension` in `Text.Pandoc.`

**Comment by jgm** [comment]
Oh, sorry, I see that you're aware of `getDefaultExtensions`.

**Comment by jgm** [comment]
One option would be to have the plain writer itself set `writerExtensions = plainExtensions`.  This would make it impossible to modify the extensions on `plain`, but I think that's for the best anyway.

**Comment by mb21** [comment]
I guess this can be closed with f9df62c29fc88634f1d4f264f894265096b0985f ?

**Comment by t-8ch** [comment]
Indeed, it can. Thanks!
